Privacy.md

```diff
@@ -0,0 +1,85 @@
+# Fruit Radar Privacy Policy.
+
+This privacy policy document is to clearly state and provides information on what, how and why personal data or information is collected, used and disclosed when this software is utilized and the choices you have associated with that data.
+
+The objective of this policy document is to provide an open environment between developer and users of this software by openly stating what information the developer collects or not. By utilizing this software, it means you accept all the terms and conditions of the developer peculiar to only Fruit Radar software. This Policy applies to this software and the developer would not be held responsible for the content or privacy practices on any third party app not operated by the developer to which this software links or that links to this software, therefore it is advisable to read up the privacy policies of third party services that may be linked to this software. <hr>
+
+## Permission and Access
+
+Permissions and access are simply the device permissions the software requires to function and the access enabling this permission grants.
+To function efficiently and effectively as developed and designed, Fruit Radar will require some device permissions. The permissions are;
+
+- **Camera**
+
+The enablement of the camera permission grants the app access to your device camera to take pictures and videos. 
+
+- **Location**
+
+When the location permission is enabled the software has access to your precise location(GPS and network-based).
+
+- **Storage**
+
+Permission to access device storage allows the software read, modify or delete contents of your SD card. 
+
+The software also has some other capabilities which include;
+
+- Ability to have full network access which means the app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+- Control vibration: This app can control your device vibrator.
+
+The above permissions can be revoked at any time you desire, but revoking these permissions will cause the software to cease to function. These permissions work hand-in-hand simultaneously to ensure smooth operations and utilization of the software. This is privy to only users with devices running Android 6.0 upwards. <hr>
+
+## Crash reports sending
+
+Fruit Radar is stable and works fine but you may still experience few errors(bugs) and crashes, information about these crashes are important to me(the developer) so the issues can be fixed immediately and the software is running efficiently again. If you encounter a bug during the use of this software, please open an [issue](https://github.com/niccokunzmann/mundraub-android/issues/new) on GitHub and it will be fixed in due time. You can also access the project's repository directly from with the app to view and open new issues.
```

Comment:
> it will be fixed in due time

I do not make promises for hobby projects. Passive voice does not say where the Energy comes from. I would say 

> people will come along who maybe also encounter this bug and we can find a solution together. You are always allowed to fix bugs and change the software yourself and we will help you if you get in contact.